src/gluonts/ext/statsforecast.py

```diff
@@ -102,13 +103,42 @@ def __init__(
         self.config = ModelConfig(quantile_levels=quantile_levels)
 
     def predict_item(self, entry: DataEntry) -> QuantileForecast:
-        # TODO use also exogenous features
         kwargs = {}
         if self.config.intervals is not None:
             kwargs["level"] = self.config.intervals
 
+        length = len(entry[FieldName.TARGET])
+        covariates = None
+
+        if FieldName.FEAT_DYNAMIC_REAL in entry:
+            covariates = entry[FieldName.FEAT_DYNAMIC_REAL].T
+            length = covariates.shape[0]
+
+        if FieldName.FEAT_STATIC_REAL in entry:
+            static_real_cov = entry[FieldName.FEAT_STATIC_REAL]
+
+            repeat_static_real_cov = static_real_cov[None, :].repeat(
+                length, axis=0
+            )
+
+            if covariates is not None:
+                covariates = np.hstack([covariates, repeat_static_real_cov])
+            else:
+                covariates = repeat_static_real_cov
```

Comment:
You could append arrays to a list and then stack them in the end without the `if`

Reply:
thanks! let me fix!